pyracf/common/utilities/logger.py

```diff
@@ -191,7 +258,11 @@ def redact_request_xml(
             # <tag operation="del" />
             if f"</{xml_key}>" not in xml_string:
                 continue
-            xml_string = self.__redact_string(xml_string, match.end(), f"</{xml_key}")
+            xml_string = re.sub(
```

Comment:
Is this still necessary?

```python
match = re.search(rf"\<{xml_key}+[^>]*\>", xml_string)
if not match:
    continue
```

Reply:
A brief explanation of it's purposes might be helpful.

Reply:
no, this is no longer necessary, my mistake. It has been removed.